syntax = "proto3";

service DatabaseCall {
  rpc CheckLogin(LoginRequest) returns(LoginReply){}
}

message LoginRequest {
  string username = 1;
  string password = 2;
}

message LoginReply {
  bool correct = 1;
}

